client/packages/react-app/build/static/media/circle.2d975615.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22C17.5228 22 22 17.5228 22 12C22 9.27455 20.9097 6.80375 19.1414 5" stroke="#2F80ED" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```

Comment:
this is react-specific SVGs right? shall we remove this?

Reply:
this is all auto-generated and happens somewhere within react-scripts (created by `create-eth-app` (a variant of `create-react-app` which is what @nichanank used).

not sure where it gets created, but we can add as a todo item for later if desired.